srt-tokio/tests/message_splitting.rs

```diff
@@ -10,6 +10,7 @@ use tokio::time::sleep;
 const PACKET_SIZE: usize = 1 << 19;
 
 #[tokio::test]
+#[ignore]
```

Comment:
Could we just make the message shorter, like 15 packets worth instead of several hundred?

Reply:
I'll experiment with this